test/DynamoCoreWpfTests/DynamoViewTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Windows.Media;
+using Dynamo.Controls;
+using Dynamo.Graph.Nodes;
+using Dynamo.Graph.Workspaces;
+using Dynamo.Models;
+using Dynamo.Selection;
+using Dynamo.Utilities;
+using Dynamo.ViewModels;
+using Dynamo.Wpf.Controls;
+using Dynamo.Wpf.ViewModels.Core;
+using DynamoCoreWpfTests.Utility;
+using NUnit.Framework;
+using SharpDX.DXGI;
+
+
+namespace DynamoCoreWpfTests
+{        
+    public class DynamoViewTests : DynamoTestUIBase
+    {
+        // adapted from: NodeViewTests.cs
+
+        protected override void GetLibrariesToPreload(List<string> libraries)
+        {
+            libraries.Add("FunctionObject.ds");
+            libraries.Add("BuiltIn.ds");
+            libraries.Add("FFITarget.dll");
+        }
+
+        public override void Open(string path)
+        {
+            base.Open(path);
+
+            DispatcherUtil.DoEvents();
+        }
+
+        public override void Run()
+        {
+            base.Run();
+
+            DispatcherUtil.DoEvents();
+        }
+
+        [Test]
+        public void FooterNotificationControlTest()
```

Comment:
Can we move this test to WorkspaceGeneralOperations class or some class in DynamoCoreWPFTests instead of creating a new class.

Reply:
I tried that, but it turns out this test really wants to be inside a test class that extends : DynamoTestUIBase. Otherwise, it breaks as it requires the resource dictionaries to be loaded. 
If you let me know if there is a good candidate to place the test inside, I'd be happy to move it there!